frb_codegen/src/parser/mod.rs

```diff
@@ -16,9 +18,73 @@ use crate::method_utils::FunctionName;
 use crate::parser::ty::TypeParser;
 use crate::source_graph::Crate;
 
+use self::ty::convert_ident_str;
+
 const STREAM_SINK_IDENT: &str = "StreamSink";
 const RESULT_IDENT: &str = "Result";
 
+pub(crate) fn get_toposort(src: HashMap<String, Type>) -> HashMap<String, Type> {
+    // Some types that cannot be Handled.
+    // Filter something like `BareFn( TypeBareFn...`
+    // Filter something like `Ptr( TypePtr { star_token: Star,`
+    let mut ret: HashMap<String, Type> = src
+        .iter()
+        .filter_map(|(k, v)| match convert_ident_str(v) {
+            Some(_) => None,
+            None => Some((k.to_owned(), v.to_owned())),
+        })
```

Comment:
`.filter(|(k,v)| !convert_ident_str(v))`

you may also need `.into_iter()` if needed

Reply:
> you may also need `.into_iter()` if needed
iter() may be better, due to src being used below

Reply:
Because of the reason above, and the output of `filter (| (_k, v) | convert_ident_str (v) is_none ())` can't use `cloned()`.
> (a,b) not implment clone()
-_-||

Am I missing something?

Reply:
would be helpful to show the whole thing

looks like somewhere can use borrow?

Reply:
Close this because I can't find a way to fix it.
Maybe the type can be changed to reference, but I think it may not have much meaning.

If there's any way you can make me suddenly understand what to do...
I will thank 🙏